tests/test_coverage.py

```diff
@@ -268,7 +268,7 @@ def test_items_that_need_coverage_respects_cutoff(self):
         )
         cr = self._coverage_record(
             pool.identifier, self.provider.data_source,
-            operation=self.provider.OPERATION
+            operation=self.provider.OPERATION, collection=self.collection
```

Comment:
Here I'm bringing the circulation tests up to the standards expected by the new revision of core by adding collections to our sample CoverageRecords.